paddle/gserver/layers/Layer.h

```diff
@@ -18,7 +18,7 @@ limitations under the License. */
 #include <functional>
 #include <memory>
 #include "ModelConfig.pb.h"
-#include "paddle/function/Function.h"
+#include "paddle/function/FunctionBase.h"
```

Comment:
Make Layer.h depends on FunctionBase.h.

It could speed up compile very much, and it makes when we change something in Function library not to recompile the whole Paddle.